docs/README.md

```diff
@@ -0,0 +1,116 @@
+# Example documentation
+
+This document describes the different software components
+used by the MoveIt 2, ROS 2, Ignition Gazebo demo,
+where to find them and how they are being called.
+
+***Note:*** only the follow target is used in this document
+
+The document details:
+
+ 1. Code repositories being used in the demo
+ 1. How to prepare a Robot URDF for using Gazebo with ros2_control
+ 1. How to connect MoveIt 2 with the Gazebo ros2_control plugins
+ 1. How this example declares the target pose for the manipulation
+ 1. Other important ROS2 packages for the simulation
+ 1. Graph of runtime ROS 2 topics and nodes
+
+## Code repositories used in the demo 
+
+Repositories used:
+
+* [Panda Ign MoveIt 2](https://github.com/AndrejOrsula/panda_ign_moveit2):
+  Software packages for Franka Emika Panda Robot that enable manipulation with MoveIt 2 
+  inside Ignition Gazebo. For control, ignition_ros2_control is used.
+
+* [ROS Ignition Gazebo](https://github.com/gazebosim/ros_gz/tree/galactichttps://github.com/gazebosim/ros_gz/tree/galactic)
+  This repository holds packages that provide integration between ROS and Ignition.
+  Mainly `ros_ign_package` is being used to launch Gazebo with ROS 2 integration 
+  and `ros_ign_bridge` to convert Gz messages to ROS messages.
+
+* [Igition Gazebo ROS2 Control](https://github.com/ros-controls/gazebo_ros2_control/tree/galactic)
```

Comment:
```suggestion
* [Ignition Gazebo ROS 2 Control](https://github.com/ros-controls/gz_ros2_control)
```
I think you want gz_ros2_control (as opposed to gazebo_ros2_control for Gz-classic) here, which is renamed from ign_ros2_control.

And here I see why you might want to say Ignition or "latest" Gazebo as that package says... sigh. Maybe best to be explicit to eliminate confusion. Maybe ~~Ignition~~ Gazebo again.